### PR TITLE
Create keras_issue_template.yaml

### PR DESCRIPTION
**Pull request by sachinprasadhs**
Create issue template for the keras.io incoming issues.

### CODE REVIEW

**Comment by sachinprasadhs** [comment]
@MarkDaoust , Could you please review and let me know if any changes are required. Thanks!

**Comment by gbaned** [comment]
Hi @MarkDaoust, Can you please review this PR ? Thank you!

File: .github/ISSUE_TEMPLATE/keras_issue_template.yaml [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        
+        **Here's why we have that policy:**.
+        
+        Keras developers respond to issues. We want to focus on work that benefits the whole community, e.g., fixing bugs and adding features. Support only helps individuals. GitHub also notifies thousands of people when issues are filed. We want them to see you communicating an interesting problem.
+        
+  - type: dropdown
+    id: source
+    attributes:
+      label: Source
+      description: Tensorflow/Keras installed from
```
Comment (pcoet) [inline .github/ISSUE_TEMPLATE/keras_issue_template.yaml:37]: "Tensorflow" -> "TensorFlow"

File: .github/ISSUE_TEMPLATE/keras_issue_template.yaml [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  - type: input
+    id: Python
+    attributes:
+      label: Python version
+      description: 
+      placeholder: e.g., 3.9
+    validations:
+      required: false
+  - type: input
+    id: Gpu
```
Comment (pcoet) [inline .github/ISSUE_TEMPLATE/keras_issue_template.yaml:79]: `Gpu` -> `GPU` or `gpu`

**Review by pcoet — CHANGES_REQUESTED** [review]

**Comment by sachinprasadhs** [comment]
Made suggested changes, you can preview it here https://github.com/sachinprasadhs/keras-io/blob/patch-2/.github/ISSUE_TEMPLATE/keras_issue_template.yaml

**Review by pcoet — APPROVED** [review]

**Pull request merged**